css/about.css

```diff
@@ -15,6 +15,51 @@ Styles for an individual page
 Page Component Styles (icon grid, team member card)
 ===================================
 */
+
+.wrapper {
+	max-width: var(--mobile-wrapper-width);
+	margin: none;
+}
+
+.invis-flex {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	/* max-height: 120px; */
+}
+
+.hero-text {
+	border: 2px solid transparent;
+	width: 369px;
+	/* display: flex;
```

Comment:
Please delete comments if this is not being used

Reply:
on it!

Reply:
completed task